test-integration/programs/flexi-counter/src/processor/schedule_intent.rs

```diff
@@ -65,9 +65,12 @@ pub fn process_create_intent(
         .map(|(committee, escrow_authority)| {
             let other_accounts = vec![
                 // counter account
-                committee.clone(),
-                transfer_destination.clone(),
-                system_program.clone(),
+                committee.into(),
+                ShortAccountMeta {
+                    pubkey: *transfer_destination.key,
+                    is_writable: true, // writable on base chain
```

Comment:
I'm not sure those  comments are needed, just put a Rust do  on top of `ShortAccountMeta::is_writable` that explains that this refers to main chain.